api/src/pcapi/core/offers/api.py

```diff
@@ -560,6 +562,12 @@ def edit_educational_stock(stock: Stock, stock_data: dict) -> None:
     educational_stock_unique_booking = bookings_repository.find_unique_eac_booking_if_any(stock.id)
     if educational_stock_unique_booking:
         validation.check_stock_booking_status(educational_stock_unique_booking)
+
+        educational_booking = _find_and_update_educational_booking_confirmation_limit_datetime(
+            educational_stock_unique_booking.educationalBookingId, booking_limit_datetime
+        )
```

Comment:
Ne pourrait-on pas "juste" écrire ça : 

```suggestion
        educational_stock_unique_booking.booking.booking_limit_datetime = booking_limit_datetime
        db.session.add(educational_stock_unique_booking)
```

... et s'éviter 2 fonctions (qui semblent à usage unique) ?

Reply:
(j'ai mis à jour ma suggestion de code)

Reply:
Effectivement, je pensais pas que ca marchait :) Je change !